docs/source/conf.py

```diff
@@ -1973,6 +1973,7 @@
     "mm_flop",
     "normalize_tuple",
     "register_flop_formula",
+    "scaled_mm_flop",
```

Comment:
Adding new entry to this file and the allowlist below are NEVER ok.
If you want this function to be public, you need to properly expose and document it.
My guess is that in this case you don't want it to be public so you should prepend an `_*` before it!

Reply:
I'm following the convention of all other functions in that file. Did I miss some aspects of it or are you saying that all those functions are also "bad" and I should rather break the convention?